admin/class-purge-post-data.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Purge Post Data stored in the database that holds scan data about the posts.
+ *
+ * @package Accessibility_Checker
+ */
+
+namespace EDAC\Admin;
+
+/**
+ * Uses sql queries to get and purge post data from the database for given post ids or for custom posts by post_type string.
+ */
+class Purge_Post_Data {
+
+	/**
+	 * Purge deleted posts
+	 *
+	 * @param int $post_id ID of the post.
+	 *
+	 * @return void
+	 */
+	public static function delete_post( int $post_id ) {
+		global $wpdb;
+		$site_id    = get_current_blog_id();
```

Comment:
I know this is the way it was before but we can avoid these single-use variables in all these methods. We can also improve readability by using multiple lines of code.

Example:
```php
    $wpdb->query( 
        $wpdb->prepare( 
            "DELETE FROM %i WHERE postid = %d AND siteid = %d", 
            $wpdb->prefix . 'accessibility_checker', 
            $post_id, 
            get_current_blog_id() 
        )
    );
```